docs/references.md

```diff
@@ -419,7 +419,7 @@ the subject expected file path
 
 | Matcher             | Description                                 |
 | :------------------ | :------------------------------------------ |
-| be exist            | The file should exist.                      |
```

Comment:
Please deprecate rather than remove.

| ~~be exist~~            | The file should exist. (deprecated)                     |

Reply:
:heavy_check_mark: dbb766a5a278da9b3ac4cd5dfb78774f7668ca82